### DISCUSSION TITLE
localStorage don't getItem, but why?

### DISCUSSION THREAD

**Discussion by antoniatandler** [neuefische/questions#625] [Web] [answered]
# localStorage setItem but don't getItem

___

## bookmarks gesetzt und im local storage gespeichert 
<img width="1430" alt="Bildschirmfoto 2023-06-19 um 10 29 11" src="https://github.com/neuefische/questions/assets/128368770/85ace26e-a9f1-463e-b4e8-ece220c7ecb2">

____

## Seite neu geladen
<img width="1432" alt="Bildschirmfoto 2023-06-19 um 10 29 29" src="https://github.com/neuefische/questions/assets/128368770/a5fc791a-2d57-4696-b841-d90d32545c2f">

___

## der Code: 

<img width="1030" alt="Bildschirmfoto 2023-06-19 um 10 35 15" src="https://github.com/neuefische/questions/assets/128368770/43a26dfe-d371-4219-8f8e-8ba65a604061">
<img width="1025" alt="Bildschirmfoto 2023-06-19 um 10 35 35" src="https://github.com/neuefische/questions/assets/128368770/86023898-507f-4e68-bf0d-1f5d18ce7101">
<img width="1006" alt="Bildschirmfoto 2023-06-19 um 10 35 54" src="https://github.com/neuefische/questions/assets/128368770/bb74b0ca-451b-40fa-ad43-e159161ba148">
___

## Console bei Vercel gibt dies aus:

<img width="548" alt="Bildschirmfoto 2023-06-19 um 10 37 04" src="https://github.com/neuefische/questions/assets/128368770/e8dd58a5-5280-4c64-a590-618ad086a058">

**Comment by doemser** [comment, marked as answer]
Easiest way to handle this use-case would be to use the [useLocalStorageState Hook](https://www.npmjs.com/package/use-local-storage-state) we handled in the react localStorage session.

You can look it up in your [course repository](https://github.com/neuefische/web-ffm-23-2/blob/main/week6-next-and-react-advanced/react-with-local-storage/react-with-local-storage.md)

**Reply by antoniatandler** [reply]
thank you! with the help of johann it worked finally! 🚀

but now I have a new problem.. vercel won't deploy the new changes, but the error message don't tell me where's the problem ... or am I missing something?

<img width="1102" alt="Bildschirmfoto 2023-06-19 um 12 28 31" src="https://github.com/neuefische/questions/assets/128368770/a130604c-1f0a-49ed-b9b6-4fb4c32cfb70">

<img width="637" alt="Bildschirmfoto 2023-06-19 um 12 27 13" src="https://github.com/neuefische/questions/assets/128368770/a3d0484d-367d-4d5b-8578-b0fe7540b35f">

**Reply by doemser** [reply]
You can expand 21 lines above the error, this should give more information

**Reply by antoniatandler** [reply]
yes but ... is this really a "node"-thing?

<img width="1117" alt="Bildschirmfoto 2023-06-19 um 12 36 39" src="https://github.com/neuefische/questions/assets/128368770/df0dbe99-6d7b-457b-93d8-bc8149cdb796">

**Reply by doemser** [reply]
Your [Pull Request History](https://github.com/antoniatandler/capstone-project/pull/16) suggests that the deployment started failing again [when you installed node into the project](https://github.com/antoniatandler/capstone-project/pull/16/commits/6d37cf90af32c8fff890e8fa21cda9b76d2e1e95)

You should not need to do that, instead:
- delete the node line from your package.json file 
- run npm install again
- push your changes

**Comment by antoniatandler** [comment]
haha just now i did this and node WAS the problem 😅
